Sources/AuthFoundation/Migration/Migration.swift

```diff
@@ -0,0 +1,82 @@
+//
+// Copyright (c) 2022-Present, Okta, Inc. and/or its affiliates. All rights reserved.
+// The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+//
+// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and limitations under the License.
+//
+
+import Foundation
+
+public enum SDKVersionMigrationError: Error {
+    case cannotCreateMigrator
+    case configurationNotFound
+    case configurationInvalid
```

Comment:
add description docs for these?

Reply:
I have stories for next sprint to expand doc coverage.